damus/Views/Wallet/TransactionsView.swift

```diff
@@ -46,15 +47,15 @@ struct TransactionView: View {
                         .bold()
                         .foregroundColor(DamusColors.adaptableBlack)
                     
-                    Text("\(relativeDate)")
+                    Text(relativeDate)
                         .font(.caption)
                         .foregroundColor(Color.gray)
                 }
                 .padding(.horizontal, 10)
                 
                 Spacer()
 
-                Text("\(txOp) \(transaction.amount/1000) sats")
+                Text(verbatim: "\(txOp) \(format_msats(transaction.amount))")
```

Comment:
Uses the existing millisats formatter to render the sats amount. This also fixes issues with pluralization on the unit.